docs/docs/client-source-without-graphql.md

```diff
@@ -0,0 +1,184 @@
+import YAMLData from "../../content/client-data.yaml"
+
+const ClientYAML = () => (
+  <div style={{ maxWidth: `960px`, margin: `1.45rem` }}>
+    <h1>{YAMLData.title}</h1>
+    <div>
+      {YAMLData.content.map(data => {
+        return <div key={`content_item_${uuid.v4()}`}>{data.item}</div>
+      })}
+    </div>
+  </div>
+)
+export default ClientYAML
+```
+
+As you can see, the code above is nothing more, nothing less than a functional stateless React component, that when rendered by Gatsby, and without any extra configuration will display a page sourced from a YAML file.
+
+## JSON example
+
+In this part you'll use JSON as a datasource.
+
+### Adding the JSON content
+
+Create a folder named `content` if it doesn't exist, and then add a new file inside called `client-data.json` with the following content inside:
+
+```json
+{
+  "title": "JSON used in the client with React and Gatsby",
+  "content": [
+    {
+      "item": "Cupcake ipsum dolor. Sit amet marshmallow topping cheesecake muffin. Halvah croissant candy canes bonbon candy. Apple pie jelly beans topping carrot cake danish tart cake cheesecake. Muffin danish chocolate soufflé pastry icing bonbon oat cake. Powder cake jujubes oat cake. Lemon drops tootsie roll marshmallow halvah carrot cake."
+    },
+    {
+      "item": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    },
+    {
+      "item": 192.33
+    },
+    {
+      "item": 111111
+    }
+  ]
+}
+```
+
+### Creating the page
+
+Now that you have something that needs to be shown, all that's missing is a page to show it.
+
+Add a new file called `jsononclient.js` inside the `pages` folder with the following code:
+
+```javascript
+import React from "react"
+import uuid from "uuid"
+import JSONData from "../../content/client-data.json"
+const ClientJSON = () => (
+  <div style={{ maxWidth: `960px`, margin: `1.45rem` }}>
+    <h1>{JSONData.title}</h1>
+    <div>
+      {JSONData.content.map(data => {
+        return <div key={`content_item_${uuid.v4()}`}>{data.item}</div>
+      })}
+    </div>
+  </div>
+)
+export default ClientJSON
+```
+
+The only thing different in this case, is the file import. Instead of the YAML file, this time you're importing directly a JSON file into the page component.
+
+Once again out of the box and without any extra configuration the page will show the content.
+
+## Joining the pieces
+
+After all these steps are complete, you should have the following file and folder structure:
+
+```
+  |gatsby-client-sourcing-YAML-JSON
+    |content
+      - client-data.yaml / - client-data.json
```

Comment:
should be 2 lines

Reply:
@muescha the "/" is for an "or", because the document can go either way you can import JSON or YAML. If it makes it more clear for you i can change the "/" to an "or".
Transforming it into:
```
  |gatsby-client-sourcing-YAML-JSON
    |content
      - client-data.yaml or - client-data.json
    
```

Reply:
Since you have later also the 2 js files without an "or" it is more clear to have here to have both files 

when we assume the user do all the tutorial sections then he have both here

Reply:
I agree with @jonniebigodes that an `or` would be more clear than including both files on separate lines...it could confuse users to say they need both in this context even if they are both covered in this doc.

Reply:
@marcysutton then I would suggest to write the or because otherwise it looks like a malformed copy paste. For or is normally used the `|` in notations

```suggestion
      - client-data.yaml or client-data.json
```